packages-exp/auth-types-exp/index.d.ts

```diff
@@ -411,3 +411,18 @@ export interface UserMetadata {
   readonly creationTime?: string;
   readonly lastSignInTime?: string;
 }
+
+/**
+ * Additional user information.
+ */
+export interface AdditionalUserInfo {
+  readonly isNewUser: boolean;
+  readonly profile?: UserProfile;
```

Comment:
profile should be `UserProfile | null`, it's not optional https://firebase.google.com/docs/reference/js/firebase.auth#additionaluserinfo

Reply:
The docs are wrong on this as well. It's never set for the base info:

https://github.com/firebase/firebase-js-sdk/blob/799507101171bce76879d93ee135b12fb922ffcf/packages/auth/src/additionaluserinfo.js#L114